__tests__/cars.test.js

```diff
@@ -0,0 +1,88 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+const Car = require('../lib/models/Cars');
+
+async function createCar({ name, model }) {
```

Comment:
Is there a reason why you are defining a function here instead of using the insert method from your model?